analyses/tp53_nf1_score/00-tp53-nf1-alterations.R

```diff
@@ -183,7 +183,9 @@ rna <- readRDS(expFile)
 # subset hist for those in rna matrix
 hist_rna <- histology %>%
   filter(Kids_First_Biospecimen_ID %in% names(rna)) %>%
-  mutate(RNA_library = case_when(RNA_library == "poly-A stranded" ~ "poly-A-stranded",
+  ## add RNA-library as unknown for CPTAC samples
+  mutate(RNA_library = case_when(grepl("^C3", Kids_First_Biospecimen_ID) ~ "unknown", 
```

Comment:
I found this in the CPTAC [paper](https://www.sciencedirect.com/science/article/pii/S1535610821000507#sec5) STAR methods, which means we can say "stranded" here
>TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold